Makefile

```diff
@@ -89,7 +89,7 @@ golangci-lint:
 
 .PHONY: go-test
 go-test: ## Run the unit test
```

Comment:
this pr has many breaking changes; the mojo needs to be updated as the pr title for a more meaningful description. e.g add support to 1..N default plugins via the layout config

Reply:
Also, it will break a lot the integration with SDK which is fine, we just need to ensure that all will still compatible. c/c @estroz